site/content/en/guides/consumer/subpackages/_index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Consume a package with subpackages"
+linkTitle: "Subpackages"
+weight: 7
+type: docs
+description: >
+  Fetch a kpt package with subpackages from a remote git repository and apply its 
+  contents to a cluster
+---
+
+{{% pageinfo color="warning" %}}
+
+#### Notice: Subpackages support is available with kpt version v0.34.0+ for [cfg] commands only
+
+{{% /pageinfo %}}
+
+This guide walks you through an example to get, view, set and apply contents of an
+example kpt package with a [subpackage] in its directory tree.
+
+## Steps
+
+1. [Fetch a remote package](#fetch-a-remote-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Provide the setter values](#provide-the-setter-values)
+4. [Apply the package](#apply-the-package)
+
+## Fetch a remote package
+
+Fetch an example kpt package with [subpackage] using [kpt pkg get].
+
+### get command
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/wordpress \
+wordpress
+```
+
+## View the package contents
+
+The primary package artifacts are Kubernetes resource configuration
+(e.g. YAML files).
+
+### tree command
```

Comment:
I have comments about the structure of this in the friction log.

Reply:
Addressed them. PTAL.